_assets/js/utils/expand-accordions.js

```diff
@@ -0,0 +1,166 @@
+// Options for the observer (which mutations to observe)
+const config = { attributes: true, childList: true, subtree: true };
+
+// Grab the button that will actually toggle things:
+const openAccordionsButton = document.querySelector('#crt-page--expandaccordions');
+
+// Select the node that will be observed for mutations
+const contentNode = document.querySelector('#crt-page--content');
+
+// Grab all of the accordions and convert the Nodelist into an array:
+const getAccordions = () => {
+  const regexp = /accordion-expandable\-*/gm;
+  const accordions = Array.from(document.querySelectorAll('.expand div.usa-accordion__content')).filter(
+    (accordion) => accordion.id.match(regexp)
+  );
+  return accordions;
+};
+
+// Grab all of the accordion buttons except for the table of contents button and convert the Nodelist into an array:
+const getAccordionButtons = () => {
+  const buttons = Array.from(document.querySelectorAll('.expand button.usa-accordion__button.pa11y-skip'));
+  return buttons;
+};
+
+// Grab all of the <details> elements and convert the Nodelist into an array:
+const getDetails = () => {
+  let details = Array.from(document.querySelectorAll('details.expand'));
```

Comment:
Can be simplified to

```
const getDetails = () => Array.from(document.querySelectorAll('details.expand'));
```

Orrrr change let to const if you want to keep it as is.